Task2/Task2.java

```diff
@@ -0,0 +1,41 @@
+package Task2;
+
+import java.util.Scanner;
+
+public class Task2 {
+    public static void main(String[] args) {
+        Scanner User_Input = new Scanner(System.in);
```

Comment:
in Java the naming convention for variables is to start with small letters and to the camelCase.
https://www.techtarget.com/whatis/definition/CamelCase#:~:text=CamelCase%20is%20a%20way%20to,programming%20and%20computer%20naming%20conventions.